Perenquen

dj-thd opened the pull request:
Añadir etiqueta funcionRaw a Columna y bugfix en escapado de cadenas en atributo funcion de Columna.

dj-thd committed:
Añadido atributo funcionRaw en Columna + bugfix
Arreglado escapado de cadenas en el atributo funcion en Columna, antes de mostrar el valor en la vista.

dj-thd committed:
Arreglada errata en el código

dj-thd committed:
Fixed bug en atributo funcionRaw de Columna, cuando no se especificaba ningun campo.

jelcaf commented on fap/compiler/src/es.fap.simpleled.generator/src/templates/GTabla.groovy:
```diff
@@ -275,9 +275,13 @@ public class GTabla extends GElement{
 		}
 		else if(c.funcion != null){
-			String str = c.funcion;
+			String str = StringEscapeUtils.escapeEcmaScript(c.funcion);
```
Al escapar cadenas que contengan "/" les añada un "\", con lo que cualquier tag de cierre html daría error. ¿Qué tal si se sustituye por?

String str = org.apache.commons.lang3.StringEscapeUtils.escapeJava(c.funcion).replace("'","\'");

dj-thd committed:
Solucionado bug
(escapeEcmaScript escapaba el caracter '/', que imposibilitaba cierre de etiquetas HTML)
